apps/src/templates/census/CensusMap.jsx

```diff
@@ -224,14 +224,23 @@ export default class CensusMap extends Component {
       });
       _this.map.addLayer({
         id: 'census-schools-teaching-cs',
-        type: 'symbol',
+        type: 'circle',
         source: _this.props.tileset,
         'source-layer': 'census',
         layout: {
-          'icon-image': 'marker-15-green',
-          'icon-allow-overlap': true,
-          // Increase the icon size as we zoom in
-          'icon-size': ['interpolate', ['linear'], ['zoom'], 1, 0.7, 14, 1.6],
+          visibility: 'visible',
+        },
+        paint: {
+          'circle-radius': 4,
+          'circle-color': [
+            'match',
+            ['get', 'teaches_cs'],
+            ['YES', 'Y', 'HISTORICAL_YES', 'HY'],
+            '#0093A4',
+            'white',
+          ],
+          'circle-stroke-width': 0.5,
+          'circle-stroke-color': '#000000',
```

Comment:
I updated this to match how the 'NO' icons are created in lines 196-215 (small circles rather than the map pointer icons) so that we could adjust the color